e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -222,6 +222,46 @@
               "createOptions": ""
             }
           },
+          "directMethodSender3": {
```

Comment:
Maybe we should call this edgeAgentDirectMethodSender or something like that?

Reply:
we could do `edgeAgentDirectMethodSender` but we decided to do the enumeration for the test module instead (at least for now).

Reply:
I think Varun's suggestion is doable, can you update in this PR?